app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BusinessEntityController.scala

```diff
@@ -354,6 +378,30 @@ class BusinessEntityController @Inject()(
     getNext(businessEntityJourney)(eori)
   }
 
+  private def sendEmail(
```

Comment:
This might be better expressed as two different methods that take arguments for the business entity and lead respectively.

Reply:
Next Pr will be my last one regarding the emails, will it be okay if i try to do this in that one?

Reply:
We need to send email to both Business entity and Lead at the same time , I don't think we can separate that functionality

Reply:
You're making two separate calls anyway so I don't think it's a massive deal.